addons/gaea/resources/gaea_data_migration.gd

```diff
@@ -0,0 +1,134 @@
+		"uid://dfjr83x416ec4": ["uid://c1koyt7wh4c4v", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3]}], #root/vector/composition/compose_vector3.tres
+		"uid://o054c8xv8xb": ["uid://b1vu2sfwynxql", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR2]}], #root/vector/decomposition/decompose_vector2.tres
+		"uid://evg3g607sf40": ["uid://b1vu2sfwynxql", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3]}], #root/vector/decomposition/decompose_vector3.tres
+		
+		"uid://cm0wp1if8nc6k": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR2, GaeaNodeVectorOp.Operation.Add]}], #root/vector/operations/add_vector2.tres
+		"uid://bq878twqcc5f": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3, GaeaNodeVectorOp.Operation.Add]}], #root/vector/operations/add_vector3.tres
+		"uid://d20pwbkvqkqnq": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR2, GaeaNodeVectorOp.Operation.Subtract]}], #root/vector/operations/substract_vector2.tres
+		"uid://boe1a3sogwvyw": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3, GaeaNodeVectorOp.Operation.Subtract]}], #root/vector/operations/substract_vector3.tres
+		"uid://cktjgkxfx8pyh": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR2, GaeaNodeVectorOp.Operation.Multiply]}], #root/vector/operations/multiply_vector2.tres
+		"uid://cq0gpnw7juqpk": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3, GaeaNodeVectorOp.Operation.Multiply]}], #root/vector/operations/multiply_vector3.tres
+		"uid://cgd05tlepxucw": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR2, GaeaNodeVectorOp.Operation.Divide]}], #root/vector/operations/divide_vector2.tres
+		"uid://hut3x2e74y85": ["uid://bclwjwmoudxkh", {&"enums": [GaeaNodeVectorBase.VectorType.VECTOR3, GaeaNodeVectorOp.Operation.Divide]}], #root/vector/operations/divide_vector3.tres
+	}
+
+	for idx in data.resource_uids.size():
+		var base_uid = data.resource_uids[idx]
+		
+		if data.node_data[idx].has("data"):
+			data.node_data[idx].set(&"arguments", data.node_data[idx].get("data"))
+			data.node_data[idx].erase("data")
+
+		if node_map.has(base_uid):
+			var target_data = node_map.get(base_uid)
+			if typeof(target_data) == TYPE_STRING:
+				data.resource_uids[idx] = target_data
+			else:
+				# Resource UID
+				data.resource_uids[idx] = target_data[0]
+				# Static data to set
+				if target_data.size() > 1:
+					for data_key in target_data[1]:
+						data.node_data[idx].set(data_key, target_data[1].get(data_key))
+				# Argument keys to rename
+				if target_data.size() > 2:
+					for old_key in target_data[2].keys():
+						var arguments: Dictionary = data.node_data[idx].get(&"arguments")
+						arguments.set(target_data[2].get(old_key), arguments.get(old_key))
+						arguments.erase(old_key)
+
+	data.other.set(&"save_version", 2)
```

Comment:
This should be GaeaData.CURRENT_SAVE_VERSION

Reply:
No because when we will update it to 3 we will forgot to update it here too